packages/suite-data/files/translations/cs.json

```
@@ -1491,6 +1491,25 @@
   "TR_SENT_TO_SELF": "Odesláno sobě",
   "TR_SERVICE_FEE": "Poplatek za služby",
   "TR_SERVICE_FEE_NOTE": "Poznámka: Platíte také poplatek za mining.",
+  "TR_SESSION_ERROR_PHASE_AFFILIATE_SERVERS_OFFLINE": "Služba coinjoin je dočasně nedostupná",
+  "TR_SESSION_ERROR_PHASE_CRITICAL_ERROR": "Kritická chyba, pozastavujeme coinjoin.",
+  "TR_SESSION_ERROR_PHASE_MISSING_UTXOS": "Účtu chybí mince, zkoušíme to dál",
+  "TR_SESSION_ERROR_PHASE_REGISTRATION_FAILED": "Ověření se zatím nezdařilo, opakuji",
+  "TR_SESSION_ERROR_PHASE_RETRYING_PAIRING": "Párování se zatím nezdařilo",
+  "TR_SESSION_ERROR_PHASE_SKIPPING_ROUND": "Přeskakujeme kolo",
+  "TR_SESSION_PHASE_AWAITING_CONFIRMATION": "Potvrzování dostupnosti",
+  "TR_SESSION_PHASE_AWAITING_SIGNATURES": "Čekání na podpisy",
+  "TR_SESSION_PHASE_AWAITING_TRANSACTION": "Čekání na transakci",
+  "TR_SESSION_PHASE_COIN_REGISTRATION": "Registrace mincí",
+  "TR_SESSION_PHASE_COIN_SELECTION": "Vybírání mincí",
+  "TR_SESSION_PHASE_REGISTERING_OUTPUTS": "Registrace výstupů",
+  "TR_SESSION_PHASE_ROUND_PAIRING": "Párování mincí v kole",
+  "TR_SESSION_PHASE_ROUND_SEARCH": "Hledání vhodného kola",
+  "TR_SESSION_PHASE_SENDING_SIGNATURE": "Odeslání koordinátorovi",
+  "TR_SESSION_PHASE_SIGNING_FAILED": "Podepisování se nezdařilo",
+  "TR_SESSION_PHASE_TRANSACTION_SIGNING": "Podepisování transakce",
+  "TR_SESSION_PHASE_WAITING_FOR_COORDINATOR": "Čekáme na koordinátora",
+  "TR_SESSION_PHASE_WAITING_FOR_OTHERS": "Čekáme na další účasníky",
```

```suggestion
  "TR_SESSION_PHASE_WAITING_FOR_COORDINATOR": "Čekání na koordinátora",
  "TR_SESSION_PHASE_WAITING_FOR_OTHERS": "Čekání na další účasníky",
```
To unify it with translations upper